Initial PR for countable pi-weight/pi-character (#1788)
pi-base/data: theorems/T000899.md

---
uid: T000899
if:
  P000243: true
then:
  P000244: true
---

> A countable $\pi$-base will also be a countable local $\pi$-base around any point.

Reviewer: You could also write this as in the proof https://topology.pi-base.org/theorems/T000270

Author: The difference being that we don't require the elements of the local pi-base to be nbhds of the point, unlike in first countability, so the pi-base is just verbatim a local pi-base without modification. So I don't think we need to overcomplicate it.

Reviewer: ```suggestion
A countable $\pi$-base is also a countable local $\pi$-base around any point.
```
Ok sure. Maybe just this tiny grammar thing then
